Makefile

```diff
@@ -27,8 +27,10 @@ build::
 	done
 	cd ${PACKDIR}/nodejs/ && \
 		yarn install && \
+		# pick up the local versions of these if possible.
+		(yarn link @pulumi/pulumi || true) && \
```

Comment:
when building a library, we attempt to pick up any locally built dependencies if possible.  Note the ```|| true``` bit.  if we aren't able to, that's not an issue.  we'll then just build against the version of the lib provided by ```yarn install```.